### DISCUSSION TITLE
There is a strange phenomenon in connecting with AirPods Pro.

### DISCUSSION THREAD

**Discussion by parkhongmin-926** [Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh#55] [Post-Install Help]
I'm the one who wrote yesterday. Sorry if I'm dirtying the bulletin board.

I'm trying to fix my Bluetooth.

I received Kext, and I used the method below on the Discussion board:
https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/issues/46

After that, Bluetooth turns on normally, and the connection is also made normally.

However, there seems to be a compatibility issue with the second generation of AirPods Pro.

The connection is fine. However, if you try to play the sound, it won't play.

But the interesting thing is, I went into the microphone setup after touching this and that in the setup, and when I go into the microphone setup, the sound comes from the 2nd generation of AirPods Pro with low quality.

Maybe, when the microphone is working, the sound is played in a low-quality way through another channel,

Does anyone know about this issue?

**Comment by Qonfused** [comment]
I've created a new issue #56 to track this.

**Reply by parkhongmin-926** [reply]
Thank you for your help!
